notebooks/import.py

```diff
@@ -82,12 +88,16 @@ def create_dataset(exp, process_date):
                 'Process date %s is older than previous import date %s' % (
                     process_date_dt, prior_import_date))
 
-    sql = ('INSERT INTO api_dataset (name, slug, date, display, import_start) '
-           'VALUES (%s, %s, %s, %s, %s) '
-           'RETURNING id')
+    sql = '''
+        INSERT INTO api_dataset
+            (name, slug, created_at, date, display, import_start)
+        VALUES (%s, %s, %s, %s, %s, %s)
+        RETURNING id
+    '''
     params = [
-        experiments.get(exp, {}).get('name', ''),
+        experiment_name,
         'TMP-%s' % exp,  # Store initially with a temporary prefix.
+        experiment_created_at,
         process_date_dt,
```

Comment:
@openjck The "Through..." date is the same -- it's the `process_date_dt` variable here. This just adds a new date which tracks the experiment creation date from normandy and sorts the `/datasets/` API by this date.

Reply:
Ah, okay. Thanks for the clarification.